packages/components/src/text/component.js

```diff
@@ -20,10 +31,6 @@ import useText from './hook';
  * }
  * ```
  */
-const Text = createComponent( {
-	as: 'span',
```

Comment:
This was refactored by assigning `span` as default value of `as` in the `useText` hook

Reply:
Why the refactor? It's not really equivalent to what was happening before (equivalent would be to add it directly to the `View as="span"` I think).

Reply:
I see. Does it mean that, for components using the `createComponent` function, the `as` prop is always going to be hardcoded? For example, the `Text` component will always be rendered as a `span`, even if we render it as `<Text as="p" />` ?